zoo/policies/open-agent/setup.py

```diff
@@ -59,7 +59,7 @@
         "smarts",
         "wheel",
         "maturin",
-        "python-bindings-open-agent-solver==0.0.1",
+        "python-bindings-open-agent-solver",
```

Comment:
The page shows version is still 0.0.0 https://pypi.org/project/python-bindings-open-agent-solver/#history

Revert this back and let it install the latest version.

Reply:
https://pypi.org/project/python-bindings-open-agent-solver/0.0.3/